README.md

```diff
@@ -273,14 +292,12 @@ block's partial config.
 </details>
 
 <details>
-<summary>How to compose blocks? (a.k.a. building presets)</summary>
+<summary>How to compose blocks? (AKA building presets)</summary>
```

Comment:
(In my head "AKA" looks more like online chat slang, while "a.k.a."  rather resembles common abbreviations in formal language)